lua/entities/gmod_wire_expression2/core/cl_http_cl.lua

```diff
@@ -0,0 +1,131 @@
+--[[
+
+	Clientside HTTP library
+	 - swadical 🍌
```

Comment:
I'm doing this in my free time, as a volunteer.

Having onerously stringent policies for contributions would likely increase code quality, but I fear it is detrimental to the collaborative nature of wiremod, where newer (or perhaps even older) contributors are discouraged from making submissions to the codebase.

Such policies and guidelines aim to improve code quality, where I don't see how attributions within the header of the file affects code quality. End-users are not going to experience improvements in CPU usage if the header is trimmed.

Then again, it is *your* repo, so I am complying.